### PR TITLE
updated elementwise for backends

### PR DESCRIPTION
**Pull request by debojit10**
logaddexp #2387

### CODE REVIEW

File: ivy/functional/ivy/elementwise.py [hunk 1]
```diff
@@ -3264,7 +3264,78 @@ def logaddexp(
...
+                        b=ivy.array([2., 3., 4.]))
+    >>> y = ivy.Container(a=ivy.array([1., 2., 3.]),\
+                        b=ivy.array([5., 6., 7.]))
+    >>> z = ivy.Container.logaddexp(y,x)
+    >>> print(z)
+    {
+        a: ivy.array([4.05, 5.05, 6.05]),
+        b: ivy.array([5.05, 6.05, 7.05])
+    }
+
```
Comment (RitujaPawas) [inline ivy/functional/ivy/elementwise.py:3336]: Please add instance method examples as well

File: ivy/array/elementwise.py [hunk 2]
```diff
@@ -1218,6 +1218,15 @@ def logaddexp(
             an array containing the element-wise results.
             The returned array must have a real-valued floating-point data
             type determined by :ref:`type-promotion`.
+
+        Examples
+        --------
+        >>> x = ivy.array([2., 5., 15.])
+        >>> y = ivy.array([3., 2., 4.])
+        >>> z = logaddexp(x, y)
```
Comment (RitujaPawas) [inline ivy/array/elementwise.py:1226]: Should call `ivy.logaddexp()`

File: ivy/functional/ivy/elementwise.py (hunk 1 above)
Comment (debojit10) [reply ivy/functional/ivy/elementwise.py:3351]: added them now

File: ivy/array/elementwise.py (hunk 2 above)
Comment (debojit10) [reply ivy/array/elementwise.py]: fixed it

**Comment by RitujaPawas** [comment]
Hey sorry for the delay. I will review your PR ASAP!

**Review by RitujaPawas — CHANGES_REQUESTED** [review]
Hi @debojit10  please review your examples, as they are failing on docstring test. Make sure they pass before requesting reviews. If you need any assistance, I will be happy to help!

**Comment by debojit10** [comment]
should be good to go now

**Review by RitujaPawas — CHANGES_REQUESTED** [review]
Hey @debojit10  Docstring tests are still failing for Container class examples!

**Comment by debojit10** [comment]
hi @RitujaPawas I am having trouble fixing the Docstring example, can you help me out?

**Comment by RitujaPawas** [comment]
> hi @RitujaPawas I am having trouble fixing the Docstring example, can you help me out?

Sure! let me take a look.

**Comment by VedPatwardhan** [comment]
Thanks for your reviews @RitujaPawas. Sorry for jumping in @debojit10, but I'll be reviewing this PR going forward to ease the load for @RitujaPawas. Could you please get your PR in phase with master as it seems 159 commits behind it? Feel free to re-request my review once you're done and we'll look into getting the docstring tests to pass. Thanks 🙂

**Comment by debojit10** [comment]
should be in sync with master now

File: ivy/functional/backends/numpy/elementwise.py [hunk 3]
```diff
@@ -418,7 +418,10 @@ def log2(x: np.ndarray, /, *, out: Optional[np.ndarray] = None) -> np.ndarray:
 
 @_handle_0_dim_output
 def logaddexp(
-    x1: np.ndarray, x2: np.ndarray, /, *, out: Optional[np.ndarray] = None
```
Comment (VedPatwardhan) [inline ivy/functional/backends/numpy/elementwise.py:421]: Please revert the change made here with the parameters.

File: ivy/functional/backends/torch/elementwise.py [hunk 4]
```diff
@@ -486,7 +486,10 @@ def abs(
 
 
 def logaddexp(
-    x1: torch.Tensor, x2: torch.Tensor, /, *, out: Optional[torch.Tensor] = None
```
Comment (VedPatwardhan) [inline ivy/functional/backends/torch/elementwise.py:489]: Please revert the change made here with the parameters.

File: ivy/container/elementwise.py [hunk 5]
```diff
@@ -4469,6 +4484,21 @@ def logaddexp(
             a container containing the element-wise results. The returned container
             must have a real-valued floating-point data type determined
             by :ref:`type-promotion`.
+
+        Examples
+        --------
+        Using :code:`ivy.Container` input:
+        
+        >>> x = ivy.Container(a=ivy.array([4, 5, 6]),\
```
Comment (VedPatwardhan) [inline ivy/container/elementwise.py:4492]: Please update this example to have all values as floats.

File: ivy/container/elementwise.py [hunk 6]
```diff
@@ -4413,6 +4413,21 @@ def static_logaddexp(
             a container containing the element-wise results. The returned container
             must have a real-valued floating-point data type determined
             by :ref:`type-promotion`.
+
+        Examples
+        --------
+        Using :code:`ivy.Container` input:
+
+        >>> x = ivy.Container(a=ivy.array([4, 5, 6]),\
```
Comment (VedPatwardhan) [inline ivy/container/elementwise.py:4421]: Please update this example to have all values as floats.

**Review by VedPatwardhan — CHANGES_REQUESTED** [review]
Hi @debojit10, thanks for making the changes. I have suggested a few more of them in order to get the docstring tests to pass. Feel free to re-request my review once you're done with them.
Thanks 🙂

**Review by debojit10 — COMMENTED** [review]
added the changes

**Comment by debojit10** [comment]
Thanks for the review @VedPatwardhan docstring test ran successfully in local environment

File: ivy/container/elementwise.py (hunk 5 above)
Comment (debojit10) [reply ivy/container/elementwise.py]: changed all values to floats

File: ivy/functional/backends/torch/elementwise.py (hunk 4 above)
Comment (debojit10) [reply ivy/functional/backends/torch/elementwise.py]: reverted the change

File: ivy/functional/backends/numpy/elementwise.py (hunk 3 above)
Comment (debojit10) [reply ivy/functional/backends/numpy/elementwise.py]: reverted the change

**Review by VedPatwardhan — APPROVED** [review]
lgtm! Thanks for contributing 🙂
